Dockerfile.compute-node

```diff
@@ -48,7 +48,20 @@ RUN cd postgres && \
     echo 'trusted = true' >> /usr/local/pgsql/share/extension/pgrowlocks.control && \
     echo 'trusted = true' >> /usr/local/pgsql/share/extension/pgstattuple.control && \
     echo 'trusted = true' >> /usr/local/pgsql/share/extension/refint.control && \
-    echo 'trusted = true' >> /usr/local/pgsql/share/extension/xml2.control
+    echo 'trusted = true' >> /usr/local/pgsql/share/extension/xml2.control && \
+    old_list="pg_stat_statements--1.0--1.1.sql pg_stat_statements--1.1--1.2.sql pg_stat_statements--1.2--1.3.sql pg_stat_statements--1.3--1.4.sql pg_stat_statements--1.4--1.5.sql pg_stat_statements--1.4.sql pg_stat_statements--1.5--1.6.sql"; \
```

Comment:
you can get the list of this files

```bash
find . -name "*.sql" -type f
```

Reply:
oh, I got what did you want to achieve

Reply:
yes, the solution explicitly lists files up to and including version 1.6 of pg_stat_statements and is generic for all future versions (not in old_list).

Reply:
I’d recommend to leave the comment in source code. Because, I assume, that someone can try to refactor these lines as I wanted to